pandas/tests/arithmetic/conftest.py

```diff
@@ -30,12 +30,21 @@ def zero(request):
 
 @pytest.fixture(params=[pd.Float64Index(np.arange(5, dtype='float64')),
                         pd.Int64Index(np.arange(5, dtype='int64')),
-                        pd.UInt64Index(np.arange(5, dtype='uint64'))],
+                        pd.UInt64Index(np.arange(5, dtype='uint64')),
+                        pd.RangeIndex(5)],
                 ids=lambda x: type(x).__name__)
 def idx(request):
     return request.param
 
 
+@pytest.fixture
+def tdser():
```

Comment:
Perhaps `td_series` ?  At least an underscore between `td` and `ser` would be good.

Reply:
Good idea, will do.